Basic-Recruit-Training/Person.cpp

```diff
@@ -0,0 +1,62 @@
+#include "Person.h"
+
+Person :: Person ()
+{
+
```

Comment:
Even though we set the member variables of Person in the main code, the default constructor should still assign the member variables to some default value, usually these are null values